build/paramfetch.go

```diff
@@ -83,8 +80,8 @@ func (ft *fetch) maybeFetchAsync(name string, info paramFile) {
 			return
 		}
 
-		ft.fetchLk.Lock()
-		defer ft.fetchLk.Unlock()
+		/*ft.fetchLk.Lock()
```

Comment:
why?

Reply:
Wanted to see if it would make fetching faster